app/src/lib/git/diff.ts

```diff
@@ -27,6 +27,9 @@ import { getOldPathOrDefault } from '../get-old-path'
 import { getCaptures } from '../helpers/regex'
 import { readFile } from 'fs/promises'
 import { forceUnwrap } from '../fatal-error'
+import { git } from '.'
```

Comment:
Let's import it straight from the file rather than the index to stay consistent with other imports. The index file is poorly maintained and I wonder if we shouldn't just get rid of it tbh.

```suggestion
import { git } from './core'
```